### PR TITLE
build-tools bump

### PR DESCRIPTION
**Pull request by dannimad**

### CODE REVIEW

**Review by CraigMacomber — APPROVED** [review]

File: packages/runtime/container-runtime/src/blobManager.ts [hunk 1]
```diff
@@ -565,7 +565,7 @@ export class BlobManager {
             const pathParts = route.split("/");
             assert(
                 pathParts.length === 3 && pathParts[1] === BlobManager.basePath,
-                "Invalid blob node id in used routes.",
```
Comment (tylerbutler) [inline packages/runtime/container-runtime/src/blobManager.ts:568]: These and other changes outside build-tools seem unrelated to this bump.

**Review by tylerbutler — APPROVED** [review]
